android/lib/src/main/java/com/ern/api/impl/core/LaunchConfig.java

```diff
@@ -229,4 +235,21 @@ public void setRootBackPressHandledByRN(boolean rootBackPressHandledByRN) {
     public boolean isRootBackPressHandledByRN() {
         return mRootBackPressHandledByRN;
     }
+
+    /**
+     * Set to true when you want the next fragment in the navigation stack to replace the current one
+     *
+     * @param mReplace true | false
+     */
+    public void setReplace(boolean mReplace) {
+        this.mReplace = mReplace;
```

Comment:
Minor: Method parameter name should be `replace`. You can then also shorten `this.mReplace` to just `mReplace`.